src/model/CarProperty.js

```diff
@@ -0,0 +1,21 @@
+export default class CarProperty {
+  constructor() {
+    this.carArray = [];
+  }
+
+  async makeCarArray(carNames) {
+    const carNameArray = CarProperty.splitStringMakeArray(carNames);
+    carNameArray.map((carName) => {
+      this.carArray.push({ name: carName, distance: 0 });
+    });
```

Comment:
배열의 `map`은 배열을 한 바퀴 순회하면서 각 요소를 새로운 값으로 mapping하고 새로운 배열을 반환하는 배열 메서드입니다!
따라서 `this.carArray`에 직접 `push`하지 않고 새로 mapping하고 싶은 값을 반환해주면 새로운 배열을 만들어줘요.

```suggestion
    this.carArray = carNameArray.map((carName) => ({ name: carName, distance: 0 }));
```